pkgs/development/python-modules/xmlsec/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, fetchFromGitHub, buildPythonPackage, pkgconfig, lxml, pkgs }:
```

Comment:
We avoid passing in `pkgs`. Instead, use `inherit (pkgs) foo;` in `python-packages.nix`.